README.md

```diff
@@ -65,126 +73,174 @@ you can then submit for evaluation to the NLP Sandbox.
 - [Node JS](https://nodejs.org/)
 - Java (required by [OpenAPITools/openapi-generator])
 
-### Start developing your own server
+### Creating a new GitHub repository
+
+Depending on the programming language-framework
```

Comment:
Maybe:

"This step will depend on your preferred programming language-framework."

Reply:
Done